src/chunk.c

```diff
@@ -3367,29 +3302,17 @@ chunk_update_status_internal(FormData_chunk *form)
 							   scankey,
 							   1,
 							   NULL,
-							   chunk_tuple_update_status,
+							   chunk_update_metadata,
```

Comment:
we locked the tuple in the previous step with a specific tid. Have you verified if this scan gets back the locked tuple?